db/migrate/20160712043931_create_storage_profiles.rb

```diff
@@ -0,0 +1,11 @@
+class CreateStorageProfiles < ActiveRecord::Migration[5.0]
+  def change
+    create_table :storage_profiles do |t|
+      t.string :name
+      t.string :uuid
+      t.string :profile_type
```

Comment:
What is the `:profile_type` tracking?  Is it this? http://pubs.vmware.com/vsphere-60/index.jsp?topic=%2Fcom.vmware.vspsdk.apiref.doc%2Fpbm.profile.CapabilityBasedProfile.ProfileCategoryEnum.html